assembl/static2/js/app/pages/voteSession.jsx

```diff
@@ -56,23 +56,24 @@ export type Proposal = {|
 |};
 
 type Props = {
-  loading: boolean,
-  title: string,
-  subTitle: string,
-  seeCurrentVotes: boolean,
+  addGaugeVote: Function,
```

Comment:
Could you be more specific on the function type ?

Reply:
i improved the return type, this is impossible to improve the parameter types